docker/docker.go

```diff
@@ -57,3 +57,8 @@ type CgroupDockerStat struct {
 	Status      string `json:"status"`
 	Running     bool   `json:"running"`
 }
+
+type ContainerStat struct {
+	Type     string            `json:"type"`
+	Metadata map[string]string `json:"metadata"`
```

Comment:
I think we would benefit from a more explicit structure here just including things we know we'll care about, such as Name, Image, ID, ..